src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -163,7 +163,18 @@ public void patchObjection(String objectionId,
 
         // if changing status to SUBMITTED from OPEN, process the objection
         if (ObjectionStatus.SUBMITTED == objectionPatch.getStatus() && ObjectionStatus.OPEN == previousStatus) {
-            objectionProcessor.process(objection, requestId);
+            ObjectionPatch patch = new ObjectionPatch();
+
+            try {
+                objectionProcessor.process(objection, requestId, patch);
+            } catch (Exception e) {
+                logger.infoContext(requestId, "Error processing objection updating status");
```

Comment:
Should we log using errorContext here?

Reply:
Normally I would agree with you, but as the exception gets rethrown directly below and gets logged in the controller, that would result in duplicating the exception in the logs.